pkg/publish/ingress.go

```diff
@@ -78,13 +78,13 @@ func toEndpoint(pattern, domain, container, appName, appNamespace string) (strin
 		App:           appName,
 		Container:     container,
 		Namespace:     appNamespace,
-		Hash:          getAppHash(container, appName, appNamespace),
```

Comment:
Can you keep this field here and have it call hashConcat with size 8.  Otherwise there's no way to update production which is using the `Hash` field in the template pattern.

Reply:
Oh shoot, I will just keep it

Reply:
Ok, I added this back.